### PR TITLE
[ALLUXIO-1768] Remove obsolete TODO comment

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: core/server/src/main/java/alluxio/master/block/BlockMasterClientRestServiceHandler.java [hunk 1]
```diff
@@ -34,7 +34,6 @@
 @NotThreadSafe
 @Path(BlockMasterClientRestServiceHandler.SERVICE_PREFIX)
 @Produces(MediaType.APPLICATION_JSON)
-// TODO(jiri): Investigate auto-generation of REST API documentation.
```
Comment (apc999) [inline core/server/src/main/java/alluxio/master/block/BlockMasterClientRestServiceHandler.java]: @jsimsa  is this true that the feature has been complete?

**Comment by apc999** [comment]
@jsimsa  PTAL

File: core/server/src/main/java/alluxio/master/block/BlockMasterClientRestServiceHandler.java (hunk 1 above)
Comment (YunLi1988) [inline core/server/src/main/java/alluxio/master/block/BlockMasterClientRestServiceHandler.java]: LGTM

**Comment by jsimsa** [comment]
LGTM

**Pull request merged**